src/appRedux/publisherConfigSlice.ts

```diff
@@ -27,6 +36,19 @@ export const publisherConfigSlice = createSlice({
     updatePaymentMethods: create.reducer((state, action: PayloadAction<[]>) => {
       state.paymentMethods = action.payload;
     }),
+    updateGoogleRecaptcha: create.reducer(
+      (state, action: PayloadAction<Partial<GoogleRecaptchaType>>) => {
+        console.log(action.payload);
```

Comment:
do we need these logs?

Reply:
Meh. I forgot to remove them. Right now should be ok